chart/templates/_envWorker.tpl

```diff
@@ -61,4 +61,32 @@
   value: {{ .Values.parquetAndInfo.targetRevision | quote }}
 - name: PARQUET_AND_INFO_URL_TEMPLATE
   value: {{ .Values.parquetAndInfo.urlTemplate | quote }}
+# specific to the split-opt-in-out-urls-scan job runner
+- name: OPT_IN_OUT_URLS_SCAN_COLUMNS_MAX_NUMBER
+  value: {{ .Values.OPT_IN_OUT_URLS_SCAN_COLUMNS_MAX_NUMBER | quote }}
+- name: OPT_IN_OUT_URLS_SCAN_MAX_CONCURRENT_REQUESTS_NUMBER
+  value: {{ .Values.OPT_IN_OUT_URLS_SCAN_MAX_CONCURRENT_REQUESTS_NUMBER | quote }}
+- name: OPT_IN_OUT_URLS_SCAN_MAX_REQUESTS_PER_SECOND
+  value: {{ .Values.OPT_IN_OUT_URLS_SCAN_MAX_REQUESTS_PER_SECOND | quote }}
+- name: OPT_IN_OUT_URLS_SCAN_ROWS_MAX_NUMBER
+  value: {{ .Values.OPT_IN_OUT_URLS_SCAN_ROWS_MAX_NUMBER | quote }}
+- name: OPT_IN_OUT_URLS_SCAN_SPAWNING_TOKEN
+  {{- if .Values.secrets.spawningToken.fromSecret }}
+  valueFrom:
+    secretKeyRef:
+      {{- if eq .Values.secrets.spawningToken.secretName "" }}
+      name: {{ .Release.Name }}-datasets-server-app-token
+      {{- else }}
+      name: {{ .Values.secrets.spawningToken.secretName | quote }}
+      {{- end }}
+      key: OPT_IN_OUT_URLS_SCAN_SPAWNING_TOKEN
```

Comment:
Not quite sure why tests are still failing, I set the secret same as in https://github.com/huggingface/datasets-server/settings/secrets/actions

Reply:
I think you need to define the env variable from the secret in the github action, maybe with
```
  env:
    OPT_IN_OUT_URLS_SCAN_SPAWNING_TOKEN: ${{ secrets. OPT_IN_OUT_URLS_SCAN_SPAWNING_TOKEN }}
```

Reply:
I added it on https://github.com/huggingface/datasets-server/pull/1044/commits/31b93af009c52eda3c88be281e6c4bae01c09301 but looks like it is still not able to find the token, will continue investigating

Reply:
Probably, Do we have to set this token for tests as in https://github.com/huggingface/datasets-server/blob/main/services/worker/tests/conftest.py#L62?